app/javascript/components/admin/UserModal.js

```diff
@@ -178,9 +207,10 @@ class UserModal extends React.Component {
               />
             </Form.Group>
             <Form.Group controlId="notes">
-              <Form.Label className="input-label">Notes</Form.Label>
+              <Form.Label className="input-label">Notes{schema?.fields?.notes?._exclusive?.required && '*'}</Form.Label>
```

Comment:
```suggestion
              <Form.Label className="input-label">Notes{schema?.fields?.notes?._exclusive?.required && ' *'}</Form.Label>
```

Reply:
I went back and forth with this one and personally prefer it without the space (see screenshots).  @hackrm thoughts from the PO side?

<img width="512" alt="Screen Shot 2022-01-26 at 8 15 01 AM" src="https://user-images.githubusercontent.com/35042815/151169549-1038972b-81b7-4b2b-b768-1d8ddf9557a5.png">

<img width="499" alt="Screen Shot 2022-01-26 at 8 15 27 AM" src="https://user-images.githubusercontent.com/35042815/151169551-afc3a58e-15b0-4fd6-8ad1-57df742ef406.png">
 it without the space

Reply:
Same comment here, just no screenshots because notes currently isn't required